src/components/modals/StatsModal.tsx

```diff
@@ -142,6 +144,19 @@ export const StatsModal = ({
           <MigrationIntro handleMigrateStatsButton={handleMigrateStatsButton} />
         </div>
       )}
+      <hr className="mt-4 -mb-4 border-gray-500" />
+      <div className="mt-5 columns-2 items-center items-stretch justify-center text-center dark:text-white sm:mt-6">
+        <div className="mt-3 text-xs">{VIEW_VERSE_TEXT}</div>
+        <a href={referenceUrl}>
```

Comment:
Let's add `target='_blank'` to open this in a separate tab

Reply:
Added! Good call, I didnt think about that.